src/components/Staking/StakingConsiderations/index.tsx

```diff
@@ -95,23 +105,10 @@ const StakingConsiderations = ({ page }: StakingConsiderationsProps) => {
                 px={2}
                 cursor="pointer"
                 h={8}
+                p={["var(--eth-space-2)", "var(--eth-space-2)"]}
```

Comment:
Similar to above, we're inside the style props of a Chakra-UI component, which allows us to use the token names directly (in this case, `2` is the token name), instead of writing them out as css var stings. Ie, `"var(--eth-space-2)"` can just be written as `"2"`.


Also, this array syntax is for breakpoints... you've assigned "2" to both, so we can just simplify this to p="2":

```suggestion
                p="2"
```